parsons/action_kit/action_kit.py

```diff
@@ -463,3 +465,83 @@ def create_generic_action(self, page, email=None, ak_id=None, **kwargs):
                                page=page,
                                return_full_json=True,
                                **kwargs)
+
+    def bulk_upload_csv(self, import_page=None, csv_file=None, autocreate_user_fields=0):
+        """
+        Bulk upload a csv file of new users or user updates.
+        If you are uploading a table object, use bulk_upload_table instead.
+        See `ActionKit User Upload Documentation
+             <https://roboticdogs.actionkit.com/docs/manual/api/rest/uploads.html>`
+        Be careful that blank values in columns will overwrite existing data.
+
+        If you get a 500 error, try sending a much smaller file (say, one row),
+        which is more likely to return the proper 400 with a useful error message
+
+        `Args:`
+            import_page: str
+                The page to post the action. The page short name.
+            csv_file: str or buffer
+                The csv file path or a file buffer object
+                A user_id or email column is required.
+            autocreate_user_fields: bool
+                When True columns starting with "user_" will be uploaded as user fields.
+                See the `autocreate_user_fields documentation
+                  <https://roboticdogs.actionkit.com/docs/manual/api/rest/uploads.html#create-a-multipart-post-request>`
+        `Returns`:
+            dict
+                success: whether upload was successful
+                progress_url: an API URL to get progress on upload processing
+                res: requests http response object
+        """
+        import_page = check_env.check('ACTION_KIT_IMPORTPAGE', import_page)
+        if csv_file is None:
+            csv_file = check_env.check('ACTION_KIT_CSVFILE', csv_file)
```

Comment:
Is there a specific reason to include the argument as a environmental variable? That's just not something that we've done with other methods (outside of instantiating classes), so I'm curious as to why. Additionally, it's not documented.

Reply:
I guess I over-interpreted feedback from the redash PR which requested env vars:
https://github.com/move-coop/parsons/pull/159#discussion_r382195027
That said, it seemed like it might be useful if someone wanted to have a generic script that just called upload_to_csv() and then provided by-environment variable when they had a different file.  However, I get that whereas the Redash use-case was loading data, this writes it, so maybe better not to.

I'll remove it